apps/web/src/components/layouts/AppLayout/Header/Rewards/Content/index.tsx

```diff
@@ -0,0 +1,56 @@
+'use client'
+
+import { useState } from 'react'
+
+import { RewardType } from '@latitude-data/core/browser'
+import { Text } from '@latitude-data/web-ui'
+
+import { RewardItem } from './RewardItem'
+import { RewardMenu } from './RewardMenu'
+
+export function RewardsContent() {
+  const [selectedType, setSelectedType] = useState<RewardType>()
+
+  if (selectedType) {
+    return (
+      <RewardMenu
+        type={selectedType}
+        onClose={() => setSelectedType(undefined)}
+      />
+    )
+  }
```

Comment:
When is a menu and when a list of reward items? This component look is doing 2 different things

Reply:
Content is either the list of rewards, or the menu of any of them. There's a `selectedReward` variable. If it's undefined, none is selected and thus the list is shown. When you click on one, a reward is selected, and its menu is displayed instead.